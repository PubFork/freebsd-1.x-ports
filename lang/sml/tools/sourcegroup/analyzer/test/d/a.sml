structure A = let structure B = C in struct val y = D.x + B.x end end

