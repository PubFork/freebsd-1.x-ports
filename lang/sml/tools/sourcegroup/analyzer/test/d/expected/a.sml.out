source sml "d/a.sml"
  import structure C D 
  export structure A ;

